great_expectations_cloud/agent/agent.py

```diff
@@ -64,13 +64,13 @@ class GXAgent:
     """
     Run GX in any environment from GX Cloud.
 
-    To start the agent, install Python and great_expectations and run `gx-agent`.
-    The agent loads a DataContext configuration from GX Cloud, and listens for
+    To start the GX Agent, install Python and great_expectations and run `gx-agent`.
+    The GX Agent loads a DataContext configuration from GX Cloud, and listens for
     user events triggered from the UI.
     """
 
     def __init__(self: Self):
-        print("Initializing GX-Agent")
+        print("Initializing the GX Agent")
```

Comment:
Since we have the debug logs, I don't think it's necessary to also log this to the log files--but I can update that if others disagree. I thought we may want to separate user prompts/info from the debug logs.